### PR TITLE
Add Action Property to Define Type of Plugin

### PR DESCRIPTION
**Pull request by aljawaid**
This will help identify plugins better

(The new plugin will be released soon)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +64,10 @@ This website is a simple HTML static webpage to serve [Kanboard](https://kanboar
   - `true` or `false`
   - Specify whether your plugin has used any hooks
   - _Boolean type_
+  - `is_action`
```
Comment (fguillot) [inline README.md]: This bullet point seems misaligned compared to others.

Comment (aljawaid) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ This website is a simple HTML static webpage to serve [Kanboard](https://kanboar
         "has_schema": false,
         "has_overrides": false,
         "has_hooks": false,
+        "is_action": "no",
```
Comment (fguillot) [inline README.md:26]: Why is this field not a boolean like others?

Comment (aljawaid) [reply README.md:26]: Because we need `yes` `no' and 'both' as answers

**Pull request merged**